docs/RELAY.md

```diff
@@ -0,0 +1,41 @@
+# mLRS Documentation: Relay #
+
+([back to main page](../README.md))
+
+**Important: Relay functionality is experimental and currently only available in the branch 'dev-relay', pre-compiled binaries can be found in the firmware folder ([here](https://github.com/olliw42/mLRS/tree/dev-relay/firmware)).**
+
+
+mLRS allows for a relay setup in which a main Tx, Rx pair is used with a secondary Tx, Rx pair to offer flexibility when locating the main Tx module. The relay setup will forward the RC channels from the radio to the flight controller while also sending the link statistics and CRSF sensors of the main Tx, Rx pair back to the radio. A relay setup is beneficial by allowing you to place the main Tx module on your antenna tracker while still being able to control the vehicle from the radio.
+
+For a relay setup is recommended to:
+- Use a lower frequency which is better suited for long range for the main Tx, Rx pair
+- Use different frequencies for the main and secondary Tx, Rx pairs to avoid interference
+- Use a faster mode for the secondary Tx, Rx pair
+
```

Comment:
just a thought which crossed my mind. The description is for when CSRF should be used between 2nd Rx and main Tx (which is what many probably want to do). A relay setup using sbus to connect 2nd Rx and main Tx would be possible with the main code. but of course with no benefit of getting the correct link stats on the radio). In some way that's really the only benefit of all of this.

Reply:
As you say, I can't see a reason that if someone is using mLRS hardware they would opt to use SBUS for the relay connection so not sure why we'd want to mention.

Reply:
:)